docs/release-notes/Percona-Everest-1.8.0-(2025-06-29).md

```diff
@@ -0,0 +1,131 @@
+
+- [EVEREST-2002](https://perconadev.atlassian.net/browse/EVEREST-2002): In the Helm upgrade flow, Percona Everest now performs a pre-check to validate CRD compatibility before proceeding with the upgrade. This helps prevent cluster breakage by ensuring that all required Custom Resource Definitions are present and compatible, improving upgrade reliability.
+
+
+- [EVEREST-2003](https://perconadev.atlassian.net/browse/EVEREST-2003): The expandable section in the **Database Overview** page has been removed for an enhanced UX.
+
+- [EVEREST-2005](https://perconadev.atlassian.net/browse/EVEREST-2005): We've added a **View DB status Details** option to the **Actions** menu on the **Overview** page, providing quicker access to database status and cluster-specific information.
+
+
+## Bug fixes
+
+- [EVEREST-1838](https://perconadev.atlassian.net/browse/EVEREST-1838): The **Edit** option for Point-in-Time Recovery (PITR) was incorrectly disabled for MySQL and MongoDB clusters, even when PITR was enabled. This issue has now been resolved.
+
+
+- [EVEREST-1890](https://perconadev.atlassian.net/browse/EVEREST-1890): During MySQL database cluster creation, the selected number of proxies was incorrectly reset to 1 in the UI. The proxy count now reflects the user's selection accurately.
+
+- [EVEREST-1895](https://perconadev.atlassian.net/browse/EVEREST-1895): Resolved an issue where the Point-in-Time Recovery (PITR) time could not be adjusted from the last successful backup. Users can now modify the PITR time as intended.
+
+- [EVEREST-1948](https://perconadev.atlassian.net/browse/EVEREST-1948): The component **Age** was previously displayed incorrectly on the **Components** tab. It now reflects the accurate age of each component.
+
+- [EVEREST-2001](https://perconadev.atlassian.net/browse/EVEREST-2001): Addressed an issue where the Disk, CPU, and Memory input fields became unresponsive or difficult to edit when large values were entered. Input behavior is now consistent and reliable across all value ranges.
+
+- [EVEREST-2030](https://perconadev.atlassian.net/browse/EVEREST-2030): Fixed an issue where users were not logged out after account deletion. The UI remained active even though the API token had been invalidated. The session is now properly terminated upon deletion.
+
+- [EVEREST-2037](https://perconadev.atlassian.net/browse/EVEREST-2037): Fixed an issue where the `Policy is being used` message appeared even when the policy was not linked to any database.
+
+- [EVEREST-2043](https://perconadev.atlassian.net/browse/EVEREST-2043): While running everestctl, setting `pmm.enabled=true` caused PMM to be deployed in the default namespace instead of the everest-system namespace. PMM is now correctly deployed in the everest-system namespace, ensuring consistency with helm install.
```

Comment:
```suggestion
- [EVEREST-2043](https://perconadev.atlassian.net/browse/EVEREST-2043): While running `everestctl install`, setting `pmm.enabled=true` caused PMM to be deployed in the default namespace instead of the everest-system namespace. PMM is now correctly deployed in the everest-system namespace, ensuring consistency with helm install.
```